ts/sagas/backendInfo.ts

```diff
@@ -22,13 +24,20 @@ const BACKEND_INFO_RETRY_INTERVAL = 60 * 60 * 10 * 1000;
 function* backendInfoWatcher(): IterableIterator<Effect> {
   const backendPublicClient = BackendPublicClient(apiUrlPrefix);
 
+  const error: Errors = [
+    {
+      context: [],
+      value: "some error occurred"
+    }
+  ];
+
   function getServerInfo(): Promise<
     t.Validation<BasicResponseType<ServerInfo>>
   > {
     return new Promise((resolve, _) =>
       backendPublicClient
         .getServerInfo({})
-        .then(resolve, () => resolve(undefined))
+        .then(resolve, () => resolve(left(error)))
```

Comment:
perhaps we can have an error that provides the rejection reason ?

```suggestion
        .then(resolve, e => resolve(left([{context: [], value: e}])))
```